about.html

```diff
@@ -0,0 +1,176 @@
+          alt="closing icon"
+          class="closing-icon"
+        />
+        <a href="index.html" class="logo-imgs"
+          ><img src="images/earth-globe.png" class="mobile-logo" alt="logo"
+        /></a>
+        <ul class="navigation">
+          <li class="menu"><a href="about.html" class="hover">About</a></li>
+          <li class="menu"><a href="#program" class="hover">Program</a></li>
+          <li class="menu"><a href="#join" class="hover">Join</a></li>
+          <li class="menu"><a href="#sponsor" class="hover">Sponsor</a></li>
+          <li class="menu"><a href="#news" class="hover">News</a></li>
+        </ul>
+      </div>
+      <div class="nav-left">
+        <img src="images/enabled.jpg" alt="Enable icon" class="enabled-menu" />
+      </div>
+      <header class="desktop-menu">
+        <div class="pre-header">
+          <div class="pre-header-flex">
+            <ion-icon name="logo-facebook" class="icon"></ion-icon>
+            <ion-icon name="logo-twitter" class="icon"></ion-icon>
+            <span class="tabs">English</span>
+            <span class="tabs">Language</span>
+            <span class="tabs">Logout</span>
+          </div>
+        </div>
+        <div class="header">
+          <a href="index.html" class="logo-imgs"
+            ><img src="images/earth-globe.png" class="header-logo" alt="logo"
+          /></a>
+          <ul class="uo-list-header">
+            <li class="menu"><a href="about.html" class="hover">About</a></li>
+            <li class="menu"><a href="#program" class="hover">Program</a></li>
+            <li class="menu"><a href="#join" class="hover">Join</a></li>
+            <li class="menu"><a href="#sponsor" class="hover">Sponsor</a></li>
+            <li class="menu"><a href="#news" class="hover">News</a></li>
+          </ul>
+          <a class="button-nav">Europe Global Summit</a>
+        </div>
+      </header>
+      <div class="headings-about">
+        <p class="sub-heading">"Hello! Europe"</p>
+        <h1 class="main-text">Europe Global Summit 2023</h1>
+      </div>
+      <div class="border-text border-text-about">
+        <p class="border-paragraph border-paragraph-about">
+          Europe Summit is the biggest gathering events in the Europe. It unites
+          all of the countries of Europe with providing speakers and
+          presentations from the rest of the World. Speakers are brought from
+          whole world to share their ideas about projects and how they
+          implemented them. They provide their history in business and their
+          projects. Most of the speakers are successful in their branch.
+        </p>
+      </div>
+      <span class="contact-span">
+        Please contact us per Email for any further questions <br />
+        about Europe Global 2023 Summit! <br
+      /></span>
+      <a href="digital@europeglobalsummit.com" class="contact-href"
+        >digital@europeglobalsummit.com</a
+      >
```

Comment:
- [ ] As requested before it is required to use the same background as the original design :+1: . [Here](https://ingdavidtm.github.io/capstone-biotechnology/images/backgroundHeaderAbout.png) you can find the required background.
- [ ] As you can see below, I think it would be better if you make your section a little bit bigger and make de description have a smaller `max-width`, to match with the template :+1: 
- [ ] Also, as you can see in the image below, there is only one title on the `about` page, so you shouldn't use "Hello Europe" here :+1:.

**Your design**

![Screenshot from 2023-02-07 08-03-23](https://user-images.githubusercontent.com/105117832/217252645-f61935b4-4027-4864-b9b8-5a47fe1e4732.png)

**Template**

![Screenshot from 2023-02-07 04-46-49](https://user-images.githubusercontent.com/105117832/217215516-59593994-9983-40cd-adf3-62497dbf6278.png)